jest.test.config.js

```diff
@@ -1,3 +1,5 @@
+const vendorsToTranspile = require('./packages/jest-preset-mc-app/vendors-to-transpile');
```

Comment:
This is to "extend" the default vendors that are shipped by default with our jest config